pandas/tests/io/test_spss.py

```diff
@@ -116,3 +116,34 @@ def test_invalid_dtype_backend():
     )
     with pytest.raises(ValueError, match=msg):
         pd.read_spss("test", dtype_backend="numpy")
+
+
+@pytest.mark.filterwarnings("ignore::pandas.errors.ChainedAssignmentError")
```

Comment:
Why is this needed?

Reply:
I add this just like other tests above in the same file. Otherwise there was a `pandas.errors.ChainedAssignmentError` (https://github.com/pandas-dev/pandas/actions/runs/6467680893/job/17558226293).

Reply:
OK thanks for referencing